qpsolvers_benchmark/test_sets/maros_meszaros_dense_posdef.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright 2022 Stéphane Caron
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Dense subset with positive definite P of the Maros-Meszaros test set."""
+
+from typing import Iterator
+
+from ..problem import Problem
+from .maros_meszaros import MarosMeszaros
+import numpy as np
+
+
+
+class MarosMeszarosDensePosdef(MarosMeszaros):
```

Comment:
We can make this class inherit from ``MarosMeszarosDense``, this way it doesn't have to redefine ``count_constraints()`` (it will be automatically inherited from the parent class).

```suggestion
class MarosMeszarosDensePosdef(MarosMeszarosDense):
```